recipes/qiime/1.9.1/meta.yaml

```diff
@@ -0,0 +1,95 @@
+about:
+  home: http://www.qiime.org
+  license: GPL2
+  summary: 'QIIME: Quantitative Insights Into Microbial Ecology'
+build:
+  number: 0
+  skip: True    # [py3k]
+package:
+  name: qiime
+  version: 1.9.1
+requirements:
+  build:
+  - python
```

Comment:
this list defines more and also much more strict dependencies than the qiime setup.py. Can you recreate the recipe with conda skeleton?